build/external-dependencies.props

```diff
@@ -0,0 +1,390 @@
+    <ExternalDependency Include="StreamJsonRpc" Version="$(StreamJsonRpcPackageVersion)" Source="$(DefaultNuGetFeed)" />
+    <ExternalDependency Include="StyleCop.Analyzers" Version="$(StyleCopAnalyzersPackageVersion)" Source="$(DefaultNuGetFeed)" Private="true" />
+    <ExternalDependency Include="System.Data.SqlClient" Version="$(Benchmarks_11_SystemDataSqlClientPackageVersion)" Source="$(DefaultNuGetFeed)" Private="true">
+      <!-- This version is used by EF's 1.1 benchmark project -->
+      <NoWarn>KRB2004</NoWarn>
+      <VariableName>Benchmarks_11_SystemDataSqlClientPackageVersion</VariableName>
+    </ExternalDependency>
+    <ExternalDependency Include="System.Interactive.Async" Version="$(SystemInteractiveAsyncPackageVersion)" Source="$(DefaultNuGetFeed)" />
+    <ExternalDependency Include="System.Management.Automation" Version="$(SystemManagementAutomationPackageVersion)" Source="$(DefaultNuGetFeed)" />
+    <ExternalDependency Include="System.Net.Http" Version="$(SystemNetHttpPackageVersion)" Source="$(DefaultNuGetFeed)" />
+    <ExternalDependency Include="System.Reactive.Linq" Version="$(SystemReactiveLinqPackageVersion)" Source="$(DefaultNuGetFeed)" />
+    <ExternalDependency Include="System.Reflection.Emit" Version="$(SystemReflectionEmitPackageVersion)" Source="$(DefaultNuGetFeed)" />
+    <ExternalDependency Include="System.Runtime.InteropServices.RuntimeInformation" Version="$(SystemRuntimeInteropServicesRuntimeInformationPackageVersion)" Source="$(DefaultNuGetFeed)" />
+    <ExternalDependency Include="WindowsAzure.Storage" Version="$(WindowsAzureStoragePackageVersion)" Source="$(DefaultNuGetFeed)" />
+    <ExternalDependency Include="xunit.abstractions" Version="$(XunitAbstractionsPackageVersion)" Source="$(DefaultNuGetFeed)" />
+    <ExternalDependency Include="xunit.analyzers" Version="$(XunitAnalyzersPackageVersion)" Source="$(DefaultNuGetFeed)" />
+    <ExternalDependency Include="xunit.assert" Version="$(XunitAssertPackageVersion)" Source="$(DefaultNuGetFeed)" />
+    <ExternalDependency Include="xunit.core" Version="$(XunitCorePackageVersion)" Source="$(DefaultNuGetFeed)" />
+    <ExternalDependency Include="xunit.extensibility.core" Version="$(XunitExtensibilityCorePackageVersion)" Source="$(DefaultNuGetFeed)" />
+    <ExternalDependency Include="xunit.runner.visualstudio" Version="$(XunitRunnerVisualstudioPackageVersion)" Source="$(DefaultNuGetFeed)" Private="true" />
+    <ExternalDependency Include="xunit" Version="$(XunitPackageVersion)" Source="$(DefaultNuGetFeed)" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(DotNetRestoreSources)'!=''">
+    <ExternalDependency
+      Update="@(ExternalDependency->WithMetadataValue('Source', '$(DotNetCoreFeed)'));@(ExternalDependency->WithMetadataValue('Source', '$(RoslynFeed)'))"
+      Source="$(DotNetRestoreSources);%(ExternalDependency.Source)" />
```

Comment:
Thoughts on this - we'd have to modify DownloadPackage to support multiple (semi-colon delimited) sources per dependency.